sandbox_externalkey_unsupported.go

```diff
@@ -0,0 +1,42 @@
+// +build !linux
+
+package libnetwork
+
+import (
+	"io"
+	"net"
+)
+
+// processSetKeyReexec is a private function that must be called only on an reexec path
+// It expects 3 args { [0] = "libnetwork-setkey", [1] = <container-id>, [2] = <controller-id> }
+// It also expects libcontainer.State as a json string in <stdin>
+// Refer to https://github.com/opencontainers/runc/pull/160/ for more information
+func processSetKeyReexec() {
+}
+
+// SetExternalKey provides a convenient way to set an External key to a sandbox
+func SetExternalKey(controllerID string, containerID string, key string) error {
+	return nil
```

Comment:
@mrjana there's a problem with https://github.com/docker/libnetwork/blob/master/controller.go#L200 because if we now returns an error something can break if some Docker code path come across this and `controller.go` seems pretty platform agnostic, WDYT?